cdap-runtime-spi/src/main/java/co/cask/cdap/runtime/spi/provisioner/Provisioner.java

```diff
@@ -52,9 +53,23 @@
    */
   Cluster createCluster(ProvisionerContext context) throws Exception;
 
+  /**
+   * Get the status of the cluster. If it does not exist, {@link ClusterStatus#NOT_EXISTS} should be returned.
```

Comment:
Is `NOT_EXIST` same as `TERMINATED`?

Reply:
there is no TERMINATED

Reply:
Ah, I mean - is it equivalent to when a cluster gets terminated? For instance, on EMR, when a cluster is terminated, it still exists, but with status ~ TERMINATED.

Reply:
oh I see. Yeah TERMINATED should map to NOT_EXIST

Reply:
👍 
LGTM